### PR TITLE
TCD-2648: Adds storage sampling docs

### PR DESCRIPTION
**Pull request by lysannep**
### Description
Adds storage sampling under project setup. 

### Motivation and Context
[TCD-2648](https://saucedev.atlassian.net/browse/TCD-2648)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

[TCD-2648]: https://saucedev.atlassian.net/browse/TCD-2648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/error-reporting/project-setup/storage-sampling.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+id: storage-sampling
+title: Storage Sampling
+sidebar_label: Storage Sampling
+description: Learn how to configure storage sampling for your Backtrace instance to limit disk usage.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Popular games or applications can see volumes of 1,000,000 submitted crashes a week with average attached object files (i.e. dumps or attachments) ranging from 15-20MB. Storing all these objects can easily fill up 17.5TB of storage every week, or 200TB for 90 days retention of data.
```
Comment (aliciakrebs) [inline docs/error-reporting/project-setup/storage-sampling.md:10]:
```suggestion
Popular games or applications can see volumes of 1,000,000 submitted crashes a week with average attached object files (i.e. dumps or attachments) ranging from 15-20 MB. Storing all these objects can easily fill up 17.5 TB of storage every week, or 200 TB for 90 days retention of data.
```

File: docs/error-reporting/project-setup/storage-sampling.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+When a fingerprint has large numbers of objects, storing most objects beyond a certain amount, generally a few dozen, adds little value. Backtrace offers storage sampling to control object storage with respect to a unique combination of fingerprint and optional specified attribute values. 
+
+Storage sampling dynamically decides upon ingestion which files should be kept in long term storage and which files can be discarded after processing is complete. Most Backtrace instances have disk quota associated to them, and storage sampling provides a way to control the usage of disk depending on your teams specific needs by limiting the number of objects stored within a fingerprint.
```
Comment (aliciakrebs) [inline docs/error-reporting/project-setup/storage-sampling.md:14]:
```suggestion
Storage sampling dynamically decides upon ingestion which files should be kept in long-term storage and which files can be discarded after processing is complete. Most Backtrace instances have disk quota associated to them, and storage sampling provides a way to control the usage of disk depending on your teams specific needs by limiting the number of objects stored within a fingerprint.
```

File: docs/error-reporting/project-setup/storage-sampling.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+As of March 5, 2020, the reset interval begins when the final rate limit's bucket is filled. Note that we are considering updating this behavior so that the interval begins upon a group's first stored object. Backtrace will work with customers to enable sampling configurations that are appropriate for their instance or projects.
+:::
+
+## Configuration
+
+Storage sampling can be configured by Admin users in **Project settings** > **Project** > **Storage sampling**. 
+
+See the example configuration below:
+
+<img src={useBaseUrl('img/error-reporting/project-settings/storage-sampling-config.png')} alt="Shows an example configuration for storage sampling." width="750" />
```
Comment (aliciakrebs) [inline docs/error-reporting/project-setup/storage-sampling.md:39]: there's a typo in the UI/screenshot should be its not it's.

**Review by aliciakrebs — APPROVED** [review]

File: docs/error-reporting/project-setup/storage-sampling.md (hunk 3 above)
Comment (lysannep) [reply docs/error-reporting/project-setup/storage-sampling.md:39]: Noted. Will log a bug with the product team -- thank you!

**Pull request merged**